frontend/src/lib/components/LemonSwitch/LemonSwitch.tsx

```diff
@@ -10,8 +10,12 @@ export interface LemonSwitchProps {
     label?: string | JSX.Element
     /** Whether the switch should use the alternative primary color. */
     alt?: boolean
+    /** Whether the switch should be wrapped in an outlined block for visual distinction */
+    block?: boolean
```

Comment:
non-blocking: the variable could be renamed and make the comment unnecessary `wrapInOutlinedBlock` :)

Reply:
Agreed. Comments should always be considered *secondary* to good naming. Code should be understood when read, and a variable like "block" can mean one of 10 different things when you encounter it down the line... (block element? block the element from working? use it as a block? what block?)

Reply:
To clarify, it's really not possible to say what will happen when this code runs `block && 'LemonSwitch__wrapper--block'`

Reply:
The idea is that block-level switches will be a common design pattern (in modals, settings pages, etc.), introduced by Chris into our design design language, so I think the comment should still be useful to note the purpose of this, as just `wrapInOutlinedBlock` doesn't tell you _why_ you'd want a block-level switch instead of an inline one. (On the side, I'm a bit opposed to using imperative "doThisAndThat" props in declarative code) I agree that "block" is really ambiguous though, perhaps "outlinedBlock" gives a clearer image? Here's a PR https://github.com/PostHog/posthog/pull/8375

Reply:
Makes sense, and I'm not anti-comment. Just indeed "block" was very ambiguous here (block what?), otherwise I'd have looked past it.

Reply:
Suggestion - instead of communicating the box model for the element, focus on its prominence or emphasis. This button-like border will make it look more actionable, gives it greater visual weight, etc. It's like bolding that switch to make it more obvious. It's almost like a primary version of the switch versus a secondary.